pandunia-lekse.csv

```diff
@@ -114,7 +114,7 @@ are|||eng:area, por:spa:área, pol:areał, fra:aire|area (extent of surface)|air
 argente|mate|Ag||silver|argent|plata|prata|||银|銀||||||||||||
 argentinia|dex|AR||Argentina|Argentine||||||||||||||Argentino|Argentiina|Argentyna
 argon|mate|Ar||argon|argon||||||||||||||argono|argon|argon
-argumente|||eng:fra:argument. spa:argumento.|argument (linguistics)|argument (actant)|argumento||||论元|項 (言語学) (kou)|논항 (nonhang)|||||||||||
+argumente|||eng:fra:argument, spa:argumento|argument (linguistics)|argument (actant)|argumento||||论元|項 (言語学) (kou)|논항 (nonhang)|||||||||||
```

Comment:
yi lekse si max ko bas long. panduni genlekse tena 1, 2 o 3 silabe.

Reply:
me aha. sa we, te ko fikra "argumen"?

Reply:
me suku "argum". mome zayo uza "dokum" sa loke da "dokumente".